app/routes/calendar/$id/report-winners.tsx

```diff
@@ -402,6 +402,7 @@ function Players({
                 variant="minimal"
                 onClick={() => handlePlayerInputTypeChange(i)}
                 data-cy="change-input-type-button"
+                className={"outline-theme"}
```

Comment:
```suggestion
                className="outline-theme"
```

This is personal preference.... should lint for this probably :)